src/main/java/com/jcabi/dynamodb/maven/plugin/CreateTablesMojo.java

```diff
@@ -111,232 +80,18 @@
-            Thread.currentThread().interrupt();
-            throw new IllegalStateException(ex);
-        }
-        Logger.info(this, "Table '%s' is now ready for use", name);
-    }
-
-    /**
-     * Get json value as a long - with a backward compatibility fallback for
-     * string values.
-     *
-     * @param json JSON input
-     * @param name The element name
-     * @return The element value converted as a long
-     */
-    private long asLong(final JsonObject json, final String name) {
-        long result;
-        try {
-            result = json.getJsonNumber(name).longValue();
-        } catch (final ClassCastException ex) {
-            result = Long.parseLong(json.getString(name));
-        }
-        return result;
-    }
-
-    /**
-     * Get projection JSON element.
-     * @param json JSON input
-     * @return Projection
-     */
-    private Projection projection(final JsonObject json) {
-        final JsonObject projn = json.getJsonObject("Projection");
-        final Projection projection = new Projection()
-            .withProjectionType(projn.getString("ProjectionType"));
-        final Collection<String> nonkeyattrs = new LinkedList<String>();
-        if (projn.containsKey("NonKeyAttributes")) {
-            for (final JsonValue nonkey
-                : projn.getJsonArray("NonKeyAttributes")) {
-                nonkeyattrs.add(nonkey.toString());
-            }
-            projection.setNonKeyAttributes(nonkeyattrs);
-        }
-        return projection;
-    }
-
-    /**
-     * Get key schema elements.
-     * @param json JSON input
-     * @return Key schema elements
-     */
-    private Collection<KeySchemaElement> keySchema(final JsonObject json) {
-        final Collection<KeySchemaElement> keys =
-            new LinkedList<KeySchemaElement>();
-        final JsonArray schema = json.getJsonArray("KeySchema");
-        for (final JsonValue value : schema) {
-            final JsonObject element = (JsonObject) value;
-            keys.add(
-                new KeySchemaElement(
-                    element.getString("AttributeName"),
-                    element.getString("KeyType")
-                )
-            );
-        }
-        return keys;
-    }
-
-    /**
-     * Reads a file's contents into a JsonObject.
-     * @param file The path of the file to read
-     * @return The JSON object
-     * @throws MojoFailureException If there is an execution failure.
-     */
-    private JsonObject readJson(final String file)
-        throws MojoFailureException {
-        InputStream stream = null;
-        JsonObject json = null;
-        try {
-            stream = new FileInputStream(file);
-            json = Json.createReader(stream).readObject();
+            new Tables(
+                this.tables,
+                this.endpoint,
+                this.tcpPort(),
+                this.key,
+                this.secret
+            )
+                .create(instances);
```

Comment:
@piddubnyi can you move indentation back by 4 spaces? Please see http://www.yegor256.com/2014/10/23/paired-brackets-notation.html#fluent